### PR TITLE
Fix/exposure db conection

### PR DESCRIPTION
**Pull request by eladyashar**
* route for load exposure data by investigationId
* route for update or create exposure
* all the relevant queries and mutation for those routes

### CODE REVIEW

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 })
 
-export default exosureRoute;
\ No newline at end of file
+exposureRoute.post('/', (request: Request, response: Response) => {
+        graphqlRequest(CREATE_EXPOSURE, request.headers, { data: request.body.data })
+        .then((result: any) => response.send(result));
+})
+
+exposureRoute.post('/update', (request: Request, response: Response) => {
```
Comment (ItayBenMoshe) [inline server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:18]: I think you can use PUT instead of POST and then delete the /update in the request

**Review by ItayBenMoshe — APPROVED** [review]

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_EXPOSURE_INFO } from '../../DBService/Exposure/Query';
+import { UPDATE_EXPOSURE, CREATE_EXPOSURE } from '../../DBService/Exposure/Mutation';
 
-const exosureRoute = Router();
+const exposureRoute = Router();
 
-exosureRoute.get('/', (request: Request, response: Response) => {
-    response.send('Hello from Exposure route');
+exposureRoute.get('/:investigationId', (request: Request, response: Response) => {
```
Comment (RoeiRom) [inline server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:8]: u sure u dont get 404, last time I did it that way I got 404
so I had to change to that:

image

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by EmilyLip — APPROVED** [review]
after the MVP - move sata parsing to server

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts (hunk 2 above)
Comment (eladyashar) [reply server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:8]: its working, i dont get 404, 
Screen Shot 2020-09-13 at 12 40 13

**Pull request merged**